config/initializers/devise.rb

```diff
@@ -249,8 +249,8 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  Devise.setup do |config|
-    config.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
+  Devise.setup do |con|
+    con.omniauth :facebook, ENV["FB_ID"], ENV["FB_SECRET"],
```

Comment:
Rubocop complained about the use of the use of word `config`, that's why I changed to `con`. I changed now to `var`? (by the way, the feature with facebook login it is disabled for now)

![screen shot 2017-03-27 at 10 24 44](https://cloud.githubusercontent.com/assets/16335156/24349660/a9333fca-12d7-11e7-9bc8-2a3bb25bca4c.png)